setup.cfg

```
@@ -39,7 +39,7 @@ install_requires =
     joblib >= 0.13.0
     statsmodels >= 0.10
     pandas
-    shap >= 0.38.1, < 0.40.0
+    shap >= 0.38.1, <= 0.40.0
```

```suggestion
    shap >= 0.38.1, < 0.41.0
```

I think we'd probably be better off allowing bug-fix releases to the 0.40 release (if any) rather than capping at exactly 0.40.0.

---

done!